packages/terra-date-picker/package.json

```diff
@@ -41,6 +41,9 @@
     "terra-modal-manager": "^5.6.0",
     "terra-responsive-element": "^4.0.0"
   },
+  "devDependencies": {
+    "terra-enzyme-intl": "^3.0.0"
```

Comment:
The Jest tests could no longer resolve the path to terra-enzyme-intl so I added it as a dev dependency

Reply:
You shouldn't need to add terra-enzyme-intl as a dev dependency here. It is already added [here](https://github.com/cerner/terra-framework/blob/master/package.json#L106). Did you tried to do a clean:install before adding it here to see if it works?

Reply:
I tried installing in the datepicker package but not the entire project. I will try that now

Reply:
Yep, that fixed it: https://github.com/cerner/terra-framework/pull/504/commits/6a98f84f536470ae7926bf7fb6e772eda34f5e78